mmdet/models/roi_heads/cascade_roi_head.py

```diff
@@ -271,6 +271,11 @@ def forward_train(self,
                     if self.bbox_head[i].custom_activation:
                         cls_score = self.bbox_head[i].loss_cls.get_activation(
                             cls_score)
+
+                    # Empty proposal.
+                    if cls_score.shape[0] == 0:
+                        break
```

Comment:
Simply break here might cause different graph during training?

Reply:
This is the refining stage, no gradient is needed. It should not cause deadlock.